src/main/java/calculator/NumberParser.java

```diff
@@ -0,0 +1,34 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class NumberParser {
+
+    private static List<String> operatorList = Arrays.asList("+", "-", "*", "/");
+
+    static void checkTextNullOrEmpty(String text) {
+        if (text == null || text.isEmpty()) {
+            throw new IllegalArgumentException("숫자가 아닙니다.");
+        }
+    }
+
+    static void checkMathOperator(String text) {
+        if (!operatorList.contains(text)) {
+            throw new IllegalArgumentException("연산자가 아닙니다.");
+        }
+    }
+
+    static long parseText(String textNumber) {
+        try {
+            return (Long.parseLong(textNumber));
+        } catch (Exception e) {
+            throw new IllegalArgumentException("숫자가 아닙니다.");
```

Comment:
```suggestion
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException("숫자가 아닙니다.");
```

추상화 수준에 맞는 Exception을 사용해주세요
>https://jaehun2841.github.io/2019/03/10/effective-java-item73/#저수준-예외의-내용이-필요하다면-예외-연쇄를-사용하라